olive/cli/auto_opt.py

```diff
@@ -163,24 +152,37 @@ def run(self):
 
     def get_run_config(self, tempdir) -> Dict:
         config = deepcopy(TEMPLATE)
+        olive_config = OlivePackageConfig.load_default_config()
 
-        excluded_passes = ["OrtPerfTuning"]
-        if self.args.use_model_builder:
-            excluded_passes.append("OnnxConversion")
-        else:
-            excluded_passes.append("ModelBuilder")
+        if self.args.provider == "DmlExecutionProvider":
+            # Force the device to gpu for Direct ML provider
+            self.args.device = "gpu"
```

Comment:
Why?

Reply:
https://devblogs.microsoft.com/directx/introducing-neural-processor-unit-npu-support-in-directml-developer-preview/